velodyne_pointcloud/package.xml

```diff
@@ -31,6 +31,8 @@
   <depend>velodyne_msgs</depend>
   <depend>yaml-cpp</depend>
 
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_cmake_xmllint</test_depend>
```

Comment:
This is included in `ament_lint_common`. See https://github.com/ament/ament_lint/blob/608c41e76023c22c82c014ed0e4c755cb8b05b6a/ament_lint_common/package.xml#L24

Reply:
Fixed in c33ff77, thanks.